src/workloads/execution/2DSphereIndex.yml

```diff
@@ -0,0 +1,62 @@
+# This workload inserts ~1GB of documents in the first 2 phases, and creates 2 types of 2dsphere indexes on them in the next 2.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 500000
+    BatchSize: 10000
+    Document:
+      a: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}]
+      name: {^RandomString: {length: 1000}}
+  - Repeat: 1
+    Database: &db test
```

Comment:
You're redefining this anchor here.

Reply:
I also feel like these two loader phases can probably be combined somehow. I think they are in the same collection. Are you explicitly doing one after the other to order the data on disk?

Reply:
Thanks for all the comments / feedback. I will be collapsing all the separate workloads into one and indexing on the collection for each type of index